### PR TITLE
[WIP] Mobile Interstitial

### PR DESCRIPTION
**Pull request by adamjgrant**
https://app.asana.com/0/42816489233171/45232540266202

### CODE REVIEW

**Comment by adamjgrant** [comment]
screen shot 2015-09-03 at 10 41 11 am

**Comment by adamjgrant** [comment]
Okay, I think we're finally good on this one, @andrewconner @cvializ

File: kifi-backend/modules/shoebox/angular/src/common/directives/mobileInterstitial/mobileInterstitial.tpl.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="kf-mobile-interstitial kf-container" ng-if="show">
+  <header class="kf-container-header">
+    <h1>Download Kifi Mobile</h1>
+  </header>
+  <main class="kf-container-main">
+    <p class="kf-mobile-interstitial-banner">Kifi in the browser isn't quite ready for small screens but we do have an excellent app for your phone.</p>
+    <img src="/img/kifi-mobile.png" />
+    <a class="kf-button kf-button-large" target="_blank" ng-href="{{ app_link }}">Download the App</a>
+    <p class="kf-mobile-interstitial-secondary-action" ng-click="permanentlyHide()">
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/directives/mobileInterstitial/mobileInterstitial.tpl.html:9]: This "clickable paragraph with a same-page link" rubs me the wrong way. What do you think about replacing it with a button styled as just text for semantics?

**Comment by cvializ** [comment]
:+1: looks awesome dude

**Pull request merged**